src/web/components/Core/DisplaySecret.scss

```diff
@@ -0,0 +1,11 @@
+.key-secret-reveal {
+  .show-button,
```

Comment:
can't remember if I commented on this before, but perhaps `view-button` is a better name?